frontend/src/components/messstelle/OptionsmenueMessstelle.vue

```diff
@@ -58,5 +85,15 @@ const getContentSheetHeight = computed(() => {
     return "400px";
 });
 
-const dialog = ref(false);
+function saveChosenOptions(): void {
+    store.commit(
+        "filteroptionsMessstelle/setFilteroptions",
+        Object.assign({}, chosenOptions.value)
+    );
+    dialog.value = false;
+}
+
+function resetOptions(): void {
+    store.dispatch("filteroptionsMessstelle/resetFilteroptions");
```

Comment:
Funktioniert noch nicht richtig. Im Optionsmenü wird weiterhin der alte ausgewählte Wert angezeigt. Die Ansicht in dem MessstelleView wird aber aktualisiert